src/intersection-observer.js

```diff
@@ -124,9 +125,17 @@ export class IntersectionObserver extends Observable {
     // The second time this is called, it doesn't do much but it
     // guarantees that the receiver gets an initial intersection change
     // record.
-    listenFor(this.iframe_, 'send-intersections', () => {
+    listenFor(this.iframe_, 'send-intersections', (data, source, origin) => {
+      // This message might be from any window within the iframe, we need
+      // to keep track of which windows want to be sent updates.
+      if (!this.clientWindows_.some(entry => entry.win == source)) {
```

Comment:
Is `send-intersections` the appropriate place for this? Perhaps another event that's fired inside `drap3p()`, like `render-start`?